core/views.py

```diff
@@ -96,7 +97,7 @@ def signup(request):
                 'token': account_activation_token.make_token(user),
             })
             to_email = form.cleaned_data.get('email')
-            send_mail('Activate your account', message, 'admin@greeklink.com', [to_email], fail_silently=False)
+            send_mail('Activate your account', message, 'verify@greeklink.com', [to_email], fail_silently=False)
```

Comment:
so should announcements get sent via this email as well? or a new email

Reply:
For now just send from this email, because once we move to production we won't be using this anymore anyways